### ISSUE TITLE
Where is Masked Language Model？

### ISSUE THREAD

**Issue by kangzhiheng** [luozhouyang/transformers-keras#13]

**Comment by luozhouyang** [comment]
Hi @kangzhiheng , Masked language model is ready since v0.4.7.

Examples:

```python
from transformers_keras import BertForMaskedLanguageModel
from transformers_keras import MaskedLanguageModelDataset

# each line is a JSON
# {"sequence": "xxxxxxxxxxxxxxxx"}
dataset = MaskedLanguageModelDataset.from_jsonl_files(input_files=["/path/to/jsonl"], batch_size=32)
model = BertForMaskedLanguageModel.from_pretrained("/path/to/pretrained/bert", with_mlm=True)
model.compile(optimizer='adam')

model.fit(dataset)


```
